release-1.8/release_notes_draft.md

```diff
@@ -142,14 +149,19 @@ kind.
 
 #### CLI Changes
 
-- The kubectl rollout and rollback implementation is complete for StatefulSet.
-- The kubectl scale command will uses the Scale subresource for kinds in the
-  apps/v1beta2 group.
-- kubectl delete will no longer scale down workload API objects prior to
-  deletion. Users who depend on ordered termination for the Pods of their
-  StatefulSet’s must use kubectl scale to scale down the StatefulSet prior to
-  deletion.
-- `kubectl create configmap` and `kubectl create secret` subcommands now support the `--append-hash` flag, which enables unique yet deterministic naming for objects generated from files, e.g. via `--from-file`.
+- StatefulSet now fully supports `kubectl rollout` and `kubectl rollback`.
+<!--Not sure whether edit reflects original meaning, but original is also 
+not clear enough. Need to check w/ author. -->
+- `kubectl scale` uses the Scale subresource for kinds in the
+  apps/v1beta2 group. 
+  <!--Meaning unclear; need to check w/ author. -->
```

Comment:
`kubectl scale` uses the `Scale` subresource for the `kinds` resource in the apps/v1beta2 group. 

(Kind is a string value representing the REST resource this object represents)

Reply:
Wouldn't it be `kind` if we're calling out the literal? Also I think an example would be useful if this is a change.

Reply:
yes, updated.